### PR TITLE
Add helm

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+              value: {{ .Values.rasterCommon.storage.s3.bucket | quote }}
+            - name: S3_FORCE_PATH_STYLE
+              value: {{ .Values.rasterCommon.storage.s3.forcePathStyle | quote }}
+            - name: S3_SSL_ENABLED
+              value: {{ .Values.rasterCommon.storage.s3.sslEnabled | quote }}
+            {{- end }}
+            - name: EXTERNAL_READINESS_URL
+              value: ''
+            - name: CLASSIFICATION_OPTIONS_FILE_LOCATION
+            {{- if eq (upper .Values.env.classification.storageProvider) "FS" }}
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:90]: i think there is a support for S3 - check with @cl-shaharshavit-rnd

File: helm/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            - name: S3_ENDPOINT
+              value: {{ .Values.rasterCommon.storage.s3.endpointUrl | quote }}
+            - name: S3_BUCKET
+              value: {{ .Values.rasterCommon.storage.s3.bucket | quote }}
+            - name: S3_FORCE_PATH_STYLE
+              value: {{ .Values.rasterCommon.storage.s3.forcePathStyle | quote }}
+            - name: S3_SSL_ENABLED
+              value: {{ .Values.rasterCommon.storage.s3.sslEnabled | quote }}
+            {{- end }}
+            - name: EXTERNAL_READINESS_URL
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:87]: ?

File: helm/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            {{- if eq (upper .Values.env.classification.storageProvider) "FS" }}
+              value: /config/classification.json
+            {{- else }}
+              value: config/classification.json
+            {{- end }}
+            {{- if .Values.caSecretName }}
+            - name: REQUESTS_CA_BUNDLE
+              value: {{ printf "%s/%s" .Values.caPath .Values.caKey | quote }}
+            - name: NODE_EXTRA_CA_CERTS
+              value: {{ printf "[%s/%s]" .Values.caPath .Values.caKey | quote }}
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:99]: REQUESTS_CA_BUNDLE & NODE_EXTRA_CA_CERTS has the same value, is it valid? the only difference is "%s/%s" vs "[%s/%s]"

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: discrete-agent
+description: A Helm chart for discrete-agent service
+type: application
+version: 1.0.0
```
Comment (asafMasa) [inline helm/Chart.yaml:5]: is this the service version?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/deployment.yaml (hunk 3 above)
Comment (AnguIar) [reply helm/templates/deployment.yaml:99]: REQUEST_CA_BUNDLE is a python env
NODE_EXTRA_CA_CERTS is a nodejs env
(it's a copy-paste by the way)

File: helm/templates/deployment.yaml (hunk 2 above)
Comment (AnguIar) [reply helm/templates/deployment.yaml:87]: It's a configuration in the service: https://github.com/MapColonies/discrete-agent/blob/master/config/custom-environment-variables.json

File: helm/templates/deployment.yaml (hunk 1 above)
Comment (AnguIar) [reply helm/templates/deployment.yaml:90]: There is an `else` clause

File: helm/Chart.yaml (hunk 4 above)
Comment (AnguIar) [reply helm/Chart.yaml:5]: it's the helm version of the service

File: helm/templates/deployment.yaml (hunk 2 above)
Comment (asafMasa) [reply helm/templates/deployment.yaml:87]: so why the value is empty?

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: helm/templates/deployment.yaml (hunk 2 above)
Comment (cl-shaharshavit-rnd) [reply helm/templates/deployment.yaml:87]: I think it was meant for deploying the agent as sidecar for another service or maybe split it to http and watcher services.
either way its irrelevant for our current deployment (no value is for "stand alone" deployment)

**Review by cl-shaharshavit-rnd — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

**Pull request merged**